jwst/coron/align_refs_step.py

```diff
@@ -38,6 +41,16 @@ def process(self, target, psf):
             # Open the input psf images
             psf_model = datamodels.open(psf)
 
+            # Retrieve the lenth of the smoothing box for the filter
```

Comment:
Two issues: One is that this check should not be needed. Any `spec` parameter will always be defined. If a parameter is not given a value, and the `spec` does not define a default, the upstream config parsing will raise an exception.

This check is probably in there because there is a name mismatch. The `spec` defines `smoothing_box_length`, but here `self.smoothing_length` is queried for.

Reply:
I should never change my mind about a variable name halfway through the process..., thanks for catching this again.